components/Item/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react'
+import { View, Text, Button, TextInput } from 'react-native'
+import './index.styl'
+import { observer } from 'startupjs'
+
+export default observer(function Item(props) {
+
+  const [edit, setEdit] = useState(false)
+  const [input, setInput] = useState(props.input)
+
+  const Edit = () => {
```

Comment:
@TimmyByDeOverly please use camelCase for functions

Reply:
@TimmyByDeOverly `toggleEdit`. toggle is an appropriate name of this action.

Reply:
done